README.md

```diff
@@ -79,77 +79,87 @@ This example demonstrates how to transcribe a local audio file:
 ```swift
 import WhisperKit
 
-// Initialize WhisperKit with default settings
-Task {
-   let pipe = try? await WhisperKit()
-   let transcription = try? await pipe!.transcribe(audioPath: "path/to/your/audio.{wav,mp3,m4a,flac}")?.text
-    print(transcription)
-}
+// Initialize WhisperKit by passing the model name (WhisperKit will automatically download it):
+let pipe = try await WhisperKit(model: "tiny")
+// Transcribe the audio file
+let transcription = try await pipe.transcribe(audioPath: "path/to/your/audio.{wav,mp3,m4a,flac}")?.text
+// Print the transcription
+print(transcription)
 ```
 
 ### Model Selection
 
-WhisperKit automatically downloads the recommended model for the device if not specified. You can also select a specific model by passing in the model name:
+You have to specify the model by passing the model name:
```

Comment:
Why this change?

Reply:
Right now user has to pass either `model` and / or `mlxModel`, depending weather mlx is used or not